src/System.Management.Automation/engine/CommandPathSearch.cs

```diff
@@ -1,14 +1,14 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
@powercode, your last commit had 1 failures in `PowerShell-CI-macos`
Job Cmdlet Tests.Retrieving partial output from jobs.Receive-Job will retrieve partial output, including -Keep results
```powershell
Destination array was not long enough. Check the destination index, length, and the array's lower bounds. (Parameter 'destinationArray')
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1: line 108
108:                         $results = Receive-Job -Keep $job

```